lib/ReactViews/Map/Panels/HelpPanel/HelpPanel.jsx

```diff
@@ -71,14 +80,28 @@ class HelpPanel extends React.Component {
           css={`
             direction: ltr;
             min-width: 295px;
-            padding: 90px 20px;
+            padding: 84px 25px;
```

Comment:
This is probably better dealt with using <Spacing right={5} /> & <Spacing bottom={17} /> (factor of 5 across our spacing/padding components) so that we can scale it accordingly for when we add in the mobile ui to this system, but I'm happy to leave this as is for now - what do you think?

Reply:
From what I understand, Spacing right/bottom just adds padding to the bottom and right side of the component right? Are you saying you might extend this to include bottom+top and right+left?

Reply:
If you put spacing with `right` on the left of something it will achieve the same thing, think of it as "x / y axis" props instead of right/down?

Reply:
Ohhh right yes that makes sense!

Reply:
Slack discussion: 
> maybe you can tackle it another way, continue to use padding but load in the whole idea of "ratio(5px) * factor(17 or whatever)" through the themes..

> in fact you can already do that with box

> see `paddedHorizontally` && `paddedVertically`

& wait for Wing's branch to be merged to mobx and I'll proceed with this